examples/with-imagekit/utils/imagekit.ts

```diff
@@ -0,0 +1,55 @@
+import ImageKit from "imagekit";
+import { FilterType } from "./types";
+import { FilterEnum } from "./enum";
+import { FileObject } from "imagekit/dist/libs/interfaces";
+
+const CONFIG_OPTIONS = {
+  publicKey: process.env.NEXT_PUBLIC_PUBLIC_KEY as string,
+  privateKey: process.env.PRIVATE_KEY as string,
+  urlEndpoint: process.env.NEXT_PUBLIC_URL_ENDPOINT as string,
+};
+
+export const listFiles = async (
+  limit = 10,
+  skip = 0,
+  filterState: FilterType,
+) => {
+  const imagekit = new ImageKit(CONFIG_OPTIONS);
```

Comment:
Why re-initiate the SDK everytime the function is called?

Reply:
How expensive is this operation?

Reply:
Updated the code moved it out of function

Reply:
Discussed on call. @ankur-dwivedi will check this once more.

Reply:
It runs when the component gets built and will only be in production once.